neoload/neoload_cli_lib/filtering.py

```diff
@@ -0,0 +1,56 @@
+import re
+
+last_filter_spec = None
+last_input_params = None
+def set_filter(filter_spec, input_params):
+    global last_filter_spec, last_input_params
+    last_filter_spec = filter_spec
+    last_input_params = input_params
+def clear_filter():
+    global last_filter_spec, last_input_params
+    last_filter_spec = None
+    last_input_params = None
+
+def stuff_current_filters(params):
```

Comment:
this function creates api_filters and cli_filters... the variables and this function could be named "parse_filters". It's purpose is not to update the "params", rather to read filter string. The merge of the returned api_filters must be done by the caller

Reply:
Resolved in 94e75c2

Reply:
Resolved in 51f5d76